packages/fxa-shared/nestjs/sentry/reporting.spec.ts

```diff
@@ -14,46 +14,58 @@ describe('filterObject', () => {
 
   // Test Sentry QueryParams filtering types
   it('should filter array of key/value arrays', () => {
-    const input = [
-      ['foo', uuid.v4().replace(/-/g, '')],
-      ['baz', uuid.v4().replace(/-/g, '')],
-      ['bar', 'fred'],
-    ];
-    const expected = [
-      ['foo', FILTERED],
-      ['baz', FILTERED],
-      ['bar', 'fred'],
-    ];
+    const input = {
+      extra: [
+        ['foo', uuid.v4().replace(/-/g, '')],
```

Comment:
Q: Should we add some helper function to generate a uuid?
I think we use the `uuid.v4().replace(/-/g, '')` about 7 places.